turbopack/crates/turbo-tasks/src/vc/traits.rs

```diff
@@ -23,8 +23,8 @@ pub unsafe trait VcValueType: ShrinkToFit + Sized + Send + Sync + 'static {
 }
 
 /// A trait implemented on all values trait object references that can be put
-/// into a Value Cell ([`Vc<&dyn Trait>`][crate::Vc]).
-pub trait VcValueTrait {
+/// into a Value Cell ([`Vc<Box<dyn Trait>>`][crate::Vc]).
+pub trait VcValueTrait: Send + Sync + 'static {
```

Comment:
Making this a subtrait of `Send + Sync` means there's a few places where we no longer need to specify `VcValueTrait + Send + Sync`.

`VcValueType` is `Send + Sync + 'static`, so this just makes this trait more closely mirror that one.